core/src/test/java/org/opensearch/sql/expression/datetime/DateTimeFunctionTest.java

```diff
@@ -434,30 +434,138 @@ public void dayOfMonth() {
   public void dayOfWeek() {
     when(nullRef.type()).thenReturn(DATE);
     when(missingRef.type()).thenReturn(DATE);
-    assertEquals(nullValue(), eval(DSL.dayofweek(nullRef)));
-    assertEquals(missingValue(), eval(DSL.dayofweek(missingRef)));
+    assertEquals(nullValue(), eval(DSL.dayofweek(functionProperties, nullRef)));
+    assertEquals(missingValue(), eval(DSL.dayofweek(functionProperties, missingRef)));
 
-    FunctionExpression expression = DSL.dayofweek(DSL.literal(new ExprDateValue("2020-08-07")));
+    FunctionExpression expression = DSL.dayofweek(
+        functionProperties,
+        DSL.literal(new ExprDateValue("2020-08-07")));
```

Comment:
Try `@ParametrizedTest` and/or split into different tests and/or use `assertAll`

Reply:
Fixed in 20150cf7693164a1d1c9aeb54a7c8db02452db6d